resources/views/livewire/header.blade.php

```diff
@@ -49,35 +49,38 @@
         <a class="button button--eye" id="button_visibility_album" title="{{ Lang::get('VISIBILITY_ALBUM') }}" data-tabindex="3">
             <svg class="iconic iconic--eye"><use xlink:href="#eye"></use></svg>
         </a>
-        <a class="button button--nsfw" id="button_nsfw_album" title="{{ Lang::get('ALBUM_MARK_NSFW') }}" data-tabindex="4">
+        <a class="button" id="button_sharing_album_users" title="{{ $locale['SHARING_ALBUM_USERS'] }}" data-tabindex="4">
```

Comment:
Instead of using `$locale` can you use the `Lang` facade (see below for the other translations).

Reply:
Oh right, I simply did a copy-paste, without looking any further...

Reply:
Should be fixed know.